strider/trapi.py

```diff
@@ -193,6 +193,10 @@ def filter_information_content(
                 if (
                     curie is not None
                     and curie.information_content < information_content_threshold
+                ) or (
```

Comment:
So my assumption here is that `curie is None` means that nodenorm doesn't know about the UMLS.   I think that a better condition (even more filtering) is that OR nodenorm does know about it and it only has a UMLS ID.

Reply:
Added.